### ISSUE TITLE
QvtkWidget

### ISSUE THREAD

**Issue by latte-z** [PointCloudLibrary/pcl#2717]
## Your Environment

* Operating System and version:
* Compiler:
* PCL Version:

**Comment by lesterlo** [comment]
I have the same vtk problem that I mentioned in #2753 .

I am using the QVTKWidget to combine the PCL Visualizer with QT. [My program](https://github.com/lesterlo/PointCloud-Object-Labeller) is not 'always' crashed. There has around 50/50 chance to trigger the crash. But when it crashed, I need to reboot my mac since the desktop environment a freeze.

I tested the same program on both Linux and Mac. The program on the Linux environment is good but the program on mac isn't.

My testing environment 
Linux: ubuntu 18.04.1
Mac: MacOS 14.04.2

**Comment by latte-z** [comment]
@lesterlo Yeah, maybe is not 'always', I remember several times everything is ok, but crash next time. So I uninstall all things and reinstall pcl-1.8.1 & vtk-8.0 & qt-5.9.7. After that I never met any crash, and I think maybe something changed in qt-5.12.

**Comment by lesterlo** [comment]
@latte-z Maybe the problem is related to a linker problem? If your program linked to the right library, everything was fine. When your program linked to a wrong library, then your program may crash. 

I receive this message "Everytime" when i running the program. I don't know what's going on. :(

```bash
objc[2024]: Class FIFinderSyncExtensionHost is implemented in both /System/Library/PrivateFrameworks/FinderKit.framework/Versions/A/FinderKit (0x7fff8f7ae210) and /System/Library/PrivateFrameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride (0x127d81dc8). One of the two will be used. Which one is undefined.
```

**Comment by latte-z** [comment]
@lesterlo It’s possible. But this message I also receive everytime when select a file from disk, maybe it’s a normal phenomenon. qvtkWidget is too old, it make the whole window can’t use HiDPI mode, I decide to wait for next milestone of PCL.

**Comment by SergioRAgostinho** [comment]
> @lesterlo It’s possible. But this message I also receive everytime when select a file from disk, maybe it’s a normal phenomenon. qvtkWidget is too old, it make the whole window can’t use HiDPI mode, I decide to wait for next milestone of PCL.

We're only aiming at finishing the transition to c++14 with the next version. We're not going to be working on this problem in any realistic timeframe :/ 

If this is important for you and if you have the time and skill, consider giving it a shot at figuring out what's going on. It's easier for us to merge pull requests from the community than actually performing bug fixing ourselves, because most of the time we're dealing with other library wide problems which affect every single platform, and only on our spare time.

Anyway, thanks for reporting and keep dumping whatever you find here.

**Comment by latte-z** [comment]
@SergioRAgostinho Thanks for your reply, because I found an alternative method, and I think this is not very important for me now. I'm not very familiar with cpp development, so I can't give more useful information. Maybe you can choose to close this issues or ignore it.
